packages/combo-box/test/overlay-position.test.js

```diff
@@ -122,7 +117,7 @@ describe('overlay position', () => {
     });
   });
 
-  (isIOS ? describe.skip : describe)('overlay alignment', () => {
+  describe('overlay alignment', () => {
```

Comment:
This check is never `true` since we don't setup for iOS tests, so I just updated the suite.